apicast/bin/apicast

```diff
@@ -110,6 +110,7 @@ done
 
 main+=("daemon ${daemon};")
 main+=("worker_processes ${worker_processes};")
+main+=$(printenv | awk '$1 ~ /^(APICAST|THREESCALE)_/ {split($0,env,"="); print "env", env[1] ";"}')
```

Comment:
1. OK, not related to this PR, probably we can discuss with @3scale/support and open a separate issue if needed.

2. Agreed. The proposal for now would just to be disciplined and document the env vars in the [parameters doc](https://github.com/3scale/apicast/blob/master/doc/parameters.md) as soon as they are added/changed/deleted (which from what I can tell has been done properly lately).